ztrouy/tour-sugar
# Band members

### Purpose
Adding a small bio for each band member, and updating the click event on bands to display the bio of each band member

### Files Changed
- Added bandMembers and bandRoles properties to database Object, and added relevant exports in `database.js`
- Updated clickEvent in `bands.js` to also show the bios of all relevant band members

### To Test
Fetch and serve the branch. Confirm that the window alert when clicking on a band now shows the name and instrument for all band members, and all venues they will be playing at. Also ensure other click events still work as intended.
